llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -220,6 +220,9 @@ def Sin  : DXILOpMapping<13, unary, int_sin,
                          "Returns sine(theta) for theta in radians.">;
 def UMax : DXILOpMapping<39, binary, int_umax,
                          "Unsigned integer maximum. UMax(a,b) = a > b ? a : b">;
+def Round : DXILOpMapping<26, unary, int_round,
```

Comment:
Please consider adding this before the `def UMax` to maintain a sorted order of opcode.